com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceAmbientOcclusion.cs

```diff
@@ -164,11 +169,19 @@ internal bool Setup(ScreenSpaceAmbientOcclusionSettings featureSettings, Scripta
                 m_Material = material;
                 m_Renderer = renderer;
                 m_CurrentSettings = featureSettings;
-                renderPassEvent = featureSettings.AfterOpaque ? RenderPassEvent.AfterRenderingOpaques : RenderPassEvent.AfterRenderingGbuffer;
 
-                ScreenSpaceAmbientOcclusionSettings.DepthSource source = isRendererDeferred
-                    ? ScreenSpaceAmbientOcclusionSettings.DepthSource.DepthNormals
-                    : m_CurrentSettings.Source;
+                ScreenSpaceAmbientOcclusionSettings.DepthSource source;
+                if (isRendererDeferred)
+                {
+                    renderPassEvent = featureSettings.AfterOpaque ? RenderPassEvent.AfterRenderingOpaques : RenderPassEvent.AfterRenderingGbuffer;
```

Comment:
Not related to this PR, but question:
Does SSAO need to run before skybox? We plan to move option to copy depth after skybox so we don't cause render target switch between opaque and skybox.